jquery.flot.navigate.js

```diff
@@ -454,14 +454,14 @@ Licensed under the MIT License ~ http://threedubmedia.googlecode.com/files/MIT-L
                 }
 
                 if (d !== 0) {
-                    var navigationOffset = axis.c2p(opts.offset.below) - axis.c2p(opts.offset.below + d);
-                    opts.offset = { below: navigationOffset + (opts.offset.below || 0), above: navigationOffset + (opts.offset.above || 0) };
+                    var navigationOffsetBelow = axis.c2p(axis.p2c(axis.min) - d) - axis.c2p(axis.p2c(axis.min)),
```

Comment:
I'm not convinced.

I think you should take into account also the navigation offsets here.